connectors/cli.py

```diff
@@ -115,6 +116,11 @@ def run(args):
 
     # load config
     config = load_config(args.config_file)
+    # Precedence: CLI args >> Config Setting >> INFO
+    set_logger(
+        args.log_level or config["service"]["log_level"] or logging.INFO,
+        filebeat=args.filebeat,
+    )
 
     # just display the list of connectors
     if args.action == "list":
```

Comment:
if log level is set to `ERROR`, `bin/elastic-ingest --action list` won't work.

I think it worth changing to `print()` instead of `logger.info()`:

```
Registered connectors:
- MongoDB
- Amazon S3
- System Directory
- MySQL
- Network Drive
- Google Cloud Storage
- Azure Blob Storage
- PostgreSQL
- Oracle Database
Bye
```

This looks much better than 
```
[FMWK][22:24:42][INFO] Registered connectors:
[FMWK][22:24:42][INFO] - MongoDB
[FMWK][22:24:42][INFO] - Amazon S3
[FMWK][22:24:42][INFO] - System Directory
[FMWK][22:24:42][INFO] - MySQL
[FMWK][22:24:42][INFO] - Network Drive
[FMWK][22:24:42][INFO] - Google Cloud Storage
[FMWK][22:24:42][INFO] - Azure Blob Storage
[FMWK][22:24:42][INFO] - PostgreSQL
[FMWK][22:24:42][INFO] - Oracle Database
[FMWK][22:24:42][INFO] Bye
```

Reply:
I agree, can change in a separate PR though. @tarekziade how do you feel?

Reply:
yeah `list` is very specific, it should not pick the log level, good catch @wangch079